### DISCUSSION TITLE
Is it safe to use InfoLabel

### DISCUSSION THREAD

**Discussion by msft-grantvanhorn** [microsoft/fluentui#29183] [General] [answered]
https://react.fluentui.dev/?path=/docs/preview-components-infolabel--default

The docs specify to contact the fluent team about use of this component. It fits our needs exactly and would be great to use. Is it safe to use?

Thanks!

**Comment by sopranopillow** [comment, marked as answer]
Hello @msft-grantvanhorn, I am currently waiting for a bug in Popover to be fixed (#29185). After the bug gets fixed, I'll be moving the component to stable after one last check. For now, the API is stable enough to use, but do know it's still not recommended to be used in production until it's in stable.

**Reply by msft-grantvanhorn** [reply]
Awesome, thanks for the quick response and linking the popover issue. I'll keep track of popover discussion as well.

**Reply by sopranopillow** [reply]
quick followup, here's the PR to check for moving InfoLabel to stable: #29198

**Reply by 15MariamS** [reply]
This was helpful for me too. Thank you!
